### DISCUSSION TITLE
Plan publishing Forge in Pinokio?

### DISCUSSION THREAD

**Discussion by Shintah07** [lllyasviel/stable-diffusion-webui-forge#785] [Ideas]
Hi, I would like to ask if it's possible to publish Forge stable diffusion in Pinokio? I have very less idea about Linux, hence Pinokio. Many had their projects published there already, making installation streamlined and help organized things immensely. I hope you'll consider. Thank you.

**Comment by aolko** [comment]
you must be kidding me https://github.com/aolko/sd-webui-forge.pinokio

**Reply by Shintah07** [reply]
My apologies. What can I say, I'm most definitely high for not seeing it. I used Pinokio's in-app Discoveries to look up Forge, but nada. My ignorance(and lack of sleep) brings shame. Nevertheless, thanks.

**Reply by 6Morpheus6** [reply]
@Shintah07 You couldn't find it on the Discover page. It was removed over a week ago since it had too many issues.
But you can still install it if you use the download link aolko posted and the "Download from URL" button on the Discover page.
If you run in any issues just join [Pinokio Discord](https://discord.gg/TQdNwadtE4) and I'll help you with it. :)
